requirements.txt

```diff
@@ -0,0 +1,95 @@
+aiohttp==2.2.5 \
```

Comment:
Is this needed if you're using a Pipfile?

Reply:
Pipfile was experimental - I'll remove the pipenv related files and let you run with standard install path.